polaris.shopify.com/src/components/GlobalSearch/GlobalSearch.tsx

```diff
@@ -64,10 +64,12 @@ function captureSearchEvent(
     value: resultRank,
   };
 
-  // i honestly have no idea which one of thes is the right set up for the google analytics version we have so let's try both
-  // and keep the one that works
-  window.gtag('event', 'customSearch', customParams);
-  window.gtag('event', 'Global Search', googleParams);
+  if (process.env.NODE_ENV === 'production') {
+    // i honestly have no idea which one of thes is the right set up for the google analytics version we have so let's try both
+    // and keep the one that works
+    window.gtag('event', 'customSearch', customParams);
+    window.gtag('event', 'Global Search', googleParams);
```

Comment:
@yurm04 can we remove one of these + the comment now?

Reply:
Forgot you were off today. Going to ship this since I'm away next week and we can follow up